dotcom-rendering/src/web/lib/getCmp.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Returns the cmp object so you don't need to import it
+ *
+ * The @guardian/consent-management-platform sets the window.guCmpHotFix
+ * value to equal the `cmp` object.
+ *
+ * If this function is called before this value has been set then an error
+ * is thrown allowing us to gather stats on any race conditions that could
+ * exist with this pattern
+ *
+ * @returns the cmp object
+ */
+export const getCmp = () => {
+	if (typeof window === 'undefined') return undefined;
```

Comment:
Maybe just:

```suggestion
	if (window === undefined) return undefined;
```

Reply:
I think that will throw an error because the code will attempt to access `window`. You have to use the `typeof` to get around this

Reply:
Ah, yes, of course!

![can’t check `window`](https://user-images.githubusercontent.com/76776/152519943-d3558f03-6a94-44a9-946a-0c295fb67b97.png)